website/docs/api/directededge.mdx

```diff
@@ -23,6 +23,7 @@ Determines whether or not the provided H3 cells are neighbors.
     {label: 'Java', value: 'java'},
     {label: 'JavaScript (Live)', value: 'javascript'},
     {label: 'Python', value: 'python'},
+    {label: 'SQL', value: 'sql'},
```

Comment:
Should this say "SQL" or "DuckDB"? Because there is [h3-pg](https://github.com/zachasme/h3-pg), as well.

Reply:
I was actually hoping that Postgres, DuckDB, presto, snowflake etc could all use the same function names and so on. Taking a quick look at  Postgres extension I suppose that is not exactly how it works right now, which is unfortunate.

Reply:
I'm open to changing function names for the sake of consistency. Generally, I renamed functions from `camelCase` to `snake_case`, while adding a `h3_` prefix (except for functions that already begun with `h3_`).

Differences appear to be double `h3_h3_` prefix and `latlng` vs `lat_lng`. Are there more I'm missing?

Reply:
Those were the main differences I noticed. In DuckDB I didn't add an extension type, so there is an additional h3_latlng_to_cell_string for indexing directly to the string form instead of uint64.

Reply:
Great, I'll add the new function names in the next release, and soft-deprecate the old ones. :+1:  Just out of curiosity, is there a reason all the other SQL bindings landed on the same naming convention?

Reply:
Well, [`h3-presto`](https://github.com/foursquare/h3-presto) was [mostly written by @isaacbrodsky](https://github.com/foursquare/h3-presto/commits/main/) ;)

Snowflake's H3 support is independent, however, [but also only a subset of functionality](https://docs.snowflake.com/en/sql-reference/functions-geospatial). So we may still need to include special logic in the documentation to point out functions that don't exist in certain SQL dialects.